cfg/aws/aws_sdk_logging_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+func TestSetSDKLogLevel(t *testing.T) {
+	cases := []struct {
+		sdkLogLevelString string
+		expectedVal       aws.LogLevelType
+	}{
+		// sdkLogLevelString does not match
+		{"FOO", aws.LogOff},
+		// Wrong case.
+		{"logDEBUG", aws.LogOff},
+		// Extra space around | is not allowed.
+		{"LogDebug | LogDebugWithSigning", aws.LogOff},
```

Comment:
Actually, I think I *would* support extra spaces (thanks for including this test case!).  I can imagine customers wanting to write it that way in their config - seems more readable.

Reply:
Fixed.